cli/dcoscli/package/main.py

```diff
@@ -510,11 +510,8 @@ def _uninstall(package_name, remove_all, app_id, cli, app, skip_confirmation):
         return 0
 
     package_manager = get_package_manager()
-    err = package.uninstall(
+    package.uninstall(
```

Comment:
Why remove this error checking?

Reply:
there is no error getting returned here, but an exception can be raised and would be caught by the main function.